.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -154,8 +191,31 @@ jobs:
     container:
       image: ${{ inputs.image }}
       options: ${{ inputs.image_options }}
+    outputs:
+      run_conclusion: ${{ steps.run.conclusion }}
```

Comment:
Is this used anywhere? What's the purpose of it?

Reply:
This is not used anywhere, I added it because I was mirroring the way `sycl-linux-build.yml` handles outputting its artifact.

We could use it to cancel the running workflows if the build fails, but I wasnt a big fan of that idea so I did not implement that.

I can remove this if that's preferable.

Reply:
Up to @sarnex , @uditagarwal97 . My preference is not to introduce dead code until it becomes necessary.

Reply:
agree